packages/x-scheduler-headless/src/base-ui-copy/temporal-adapter-date-fns/TemporalAdapterDateFns.ts

```diff
@@ -87,11 +87,11 @@ const FORMATS: TemporalAdapterFormats = {
   localizedNumericDate: 'P', // Note: Day and month are padded on enUS unlike Luxon
 };
 
-// declare module '@base-ui-components/react/types' {
```

Comment:
This allow to type `SchedulerValidDate` correctly.
So far, it was `any`, so you could pass a non-processed event to a method that expected a processed event and TS did not complain :/